### DISCUSSION TITLE
This program uninstalls windows defender in such a severe way that any way other than a clean installation could bring back windows defender

### DISCUSSION THREAD

**Discussion by Sashaelfxp** [ionuttbara/windows-defender-remover#190] [General]
I just wanted to reinstall windows security to solve a bug but I've been trying to reinstall windows security for 2 hours and no video or google result has worked for me even already use securityhealthsetup this is something I wasn't prepared to do today I have so many settings and games that I don't want to lose but thanks to this "genius" now I'm forced to do a clean install of windows 11, why don't you warn that windows security no security will be able to reinstall? i thought that this was a desistalacion not a hard break, if you read this and have doubts about whether or not to use the program do not use it and better if you want to permanently disable it use windows defender control and to re-activate it you just have to open the program and activate it and not this nonsense of "script"
![missing security windows](https://github.com/user-attachments/assets/34eb4e1c-07b2-42a7-b89b-501aa13d3933)

**Comment by lowercaselowen** [comment]
all I see is a screen of black - is that your PC now? lol

**Comment by DamageInc78** [comment]
Yeah blame your blatant stupidity on the script creator.
